examples/stable_diffusion_xl/gm/helpers.py

```diff
@@ -632,7 +630,11 @@ def perform_save_locally(save_path, samples):
         base_count += 1
 
 
-def save_checkpoint(model, path, only_save_lora=False):
+def _build_lora_ckpt_path(ckpt_path):
+    return ckpt_path[: -len(".ckpt")] + "_lora.ckpt"
```

Comment:
hard to read, just to replace by `ckpt_path.replace(".ckpt", "_lora.ckpt"`)

Reply:
Updated